arangodb-net-standard/Docs/usage.md

```diff
@@ -0,0 +1,301 @@
+# ArangoDB-net-standard Usage
+
+## Quick Start
+
+The following examples can be read one after the other to form a complete working example.
+
+### Create a database
+
+```
```

Comment:
We can add `csharp` here to display the code a bit nicer, like in some other snippets below in the file.